client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js

```diff
@@ -326,19 +326,49 @@ export default class CreateOrUpdateForm extends React.Component {
                 latitude.isValid = false;
                 latitude.message = 'Latitude must not be blank';
                 formIsValid = false;
+            } else if (latitude.value.includes('°')) {
+                latitude.isValid = false;
+                latitude.message = 'Please use decimal coordinates, not degrees. ' +
+                    'To convert, input your degrees into Google Maps and copy the new numbers here.';
+                formIsValid = false;
             } else if (!validator.matches(latitude.value, latitudeRegex)) {
                 latitude.isValid = false;
                 latitude.message = 'Latitude must be valid';
                 formIsValid = false;
+            } else {
+                const latAsDouble = parseFloat(latitude.value);
+                // Alaska is the furthest north location and its latitude is approximately 71
+                // The American Samoa is the furthest south location and its latitude is approximately -14
+                if (latAsDouble > 72 || latAsDouble < -15) {
```

Comment:
@benvogler I'd appreciate you double-checking these numbers and the logic

Reply:
This is the page I mentioned last week - https://en.wikipedia.org/wiki/List_of_extreme_points_of_the_United_States

Based on that, this looks good